internal/config/config_file_test.go

```diff
@@ -150,20 +153,149 @@ func Test_parseConfigFile(t *testing.T) {
 
 func Test_ConfigDir(t *testing.T) {
 	tests := []struct {
-		envVar string
-		want   string
+		name        string
+		onlyWindows bool
+		env         map[string]string
+		output      string
 	}{
-		{"/tmp/gh", ".tmp.gh"},
-		{"", ".config.gh"},
+		{
+			name:   "no envVars",
+			output: ".config.gh",
+			env: map[string]string{
+				"AppData": "",
```

Comment:
Shouldn't this test _always_ stub out all the environment variables that the implementation relies on? Meaning:
1. HOME
2. USERPROFILE
3. XDG_CONFIG_HOME
4. GH_CONFIG_DIR

This would be to neutralize the environment that the tests are run in. #3044